dynaconf/utils/__init__.py

```diff
@@ -8,11 +8,13 @@
 from typing import Any
 from typing import Iterator
 from typing import TYPE_CHECKING
+from typing import TypedDict
 from typing import TypeVar
 
 if TYPE_CHECKING:  # pragma: no cover
     from dynaconf.base import LazySettings
     from dynaconf.base import Settings
+    from dynaconf.data_nodes import DataNode
```

Comment:
isn´t this `nodes` ?


```suggestion
    from dynaconf.nodes import DataNode
```